lib/tasks/transforms/amd/index.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
-const stew = require('broccoli-stew');
+// Why is this rule erroring here?
```

Comment:
Any ideas? It is definitely published.

Reply:
Probably because there is a package.json in this directory. You’d need to add broccoli-stew to that package.json (like with a version constraint of `*`).

Reply:
I removed broccoli-stew. I guess I need to get `'../../../broccoli/amd-shim'` in the package.json somehow.